kubemarine/kubernetes/__init__.py

```diff
@@ -241,6 +241,13 @@ def enrich_inventory(inventory: dict, cluster: KubernetesCluster) -> dict:
     preflight_errors.extend(default_preflight_errors)
     inventory["services"]["kubeadm_flags"]["ignorePreflightErrors"] = ",".join(set(preflight_errors))
 
+    # override kubeadm_kube-proxy.conntrack.min with sysctl.net.netfilter.nf_conntrack_max since they define the same kernel variable
+    version_key = utils.version_key(get_initial_kubernetes_version(inventory))
+    if version_key >= (1, 29, 0):
+        inventory["services"]["kubeadm_kube-proxy"]["conntrack"]["min"] = inventory["services"]["sysctl"]["net.netfilter.nf_conntrack_max"]
+    else:
+        inventory["services"]["kubeadm_kube-proxy"]["conntrack"]["min"] = ''
```

Comment:
More simpler will be to `del` the value, and do not change the order of enrichment functions.

`manage_primitive_values` is designed to be just after `compile_inventory`. Otherwise, it is too easy to introduce some code between `compile_inventory` and `manage_primitive_values` that will rely on, for example, plugins.install, and will not take into account that it may be a string value.

Also, I suggest to remove corresponding jinja template from defaults.yaml and one related section from `manage_primitive_values` as no longer necessary.

Reply:
Done.